static/scss/homepage.scss

```diff
@@ -888,3 +906,29 @@ footer {
 .rating-lg .icon {
   font-size: 18px;
 }
+h4.subscription-success {
+    display:none;
+    font-weight:700;
+    color:#fff;
+    font-size:.9em;
+    background:#417505;
+    padding:14px;
+    -webkit-border-radius:4px;
+    -moz-border-radius:4px;
+    -ms-border-radius:4px;
+    -o-border-radius:4px;
+    border-radius:4px;
+}
+h4.subscription-error {
```

Comment:
can you define a single class that `.subscription-success` and `.subscription-error` elements can share? there is a lot of repetition in the style definitions here. maybe one class `.subscription-result` that both elements use, and an extra `.success` and `.error` class to differentiate the two